lib/OPCUA/Open62541.pm

```diff
@@ -270,8 +270,19 @@ my @statuscodes = qw(
     STATUSCODE_BADMAXCONNECTIONSREACHED
 );
 
+my @clientstates = qw(
+    CLIENTSTATE_DISCONNECTED
+    CLIENTSTATE_WAITING_FOR_ACK
+    CLIENTSTATE_CONNECTED
+    CLIENTSTATE_SECURECHANNEL
+    CLIENTSTATE_SESSION
+    CLIENTSTATE_SESSION_DISCONNECTED
+    CLIENTSTATE_SESSION_RENEWED
+);
+
 our %EXPORT_TAGS = (
-    'all' => [ @limits, @statuscodes ],
+    'all' => [ @limits, @statuscodes, @clientstates ],
+    'clientstates' => [ @clientstates ],
```

Comment:
I called the key in singular, see 'statuscode'.  Use 'clientstate' for consistency.  Plural s for arrays is fine.